daemon/daemon_unix.go

```diff
@@ -189,6 +197,15 @@ func verifyPlatformContainerSettings(daemon *Daemon, hostConfig *runconfig.HostC
 			return warnings, fmt.Errorf("Invalid value: %v, valid memory swappiness range is 0-100.", swappiness)
 		}
 	}
+	if hostConfig.KernelMemory > 0 && !sysInfo.KernelMemory {
+		warnings = append(warnings, "Your kernel does not support kernel memory limit capabilities. Limitation discarded.")
```

Comment:
hi @moxiegirl This description was trying to be constant with other warnings in this function, if they were not good enough, maybe we can pull an separate request and change them all in once?